src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -110,12 +110,12 @@ public override Task<bool> HandleRequestAsync()
             var remoteSignOutContext = new RemoteSignOutContext(Context, Scheme, Options, message);
             await Events.RemoteSignOut(remoteSignOutContext);
 
-            if (remoteSignOutContext.HandledResponse)
+            if (remoteSignOutContext.State == EventResultState.HandleResponse)
             {
                 Logger.RemoteSignOutHandledResponse();
                 return true;
             }
-            if (remoteSignOutContext.Skipped)
+            if (remoteSignOutContext.State == EventResultState.SkipToNextMiddleware)
```

Comment:
> I don't think you've convinced us for the need for Bypass yet.

I don't need to, it's already present in the 1.0/1.1/2.0 codebase in places like `JwtBearerHandler.HandleChallengeAsync()`... but it's named `SkipToNextMiddleware()` / `HandleResponse()` names, which makes no sense because we can't do that in `HandleChallengeAsync()`.

`Bypass` doesn't introduce a new logic, it's just a better name that represents what's really happening: skipping the default logic.